Sources/PkgConfig/PkgConfig.swift

```diff
@@ -58,7 +64,7 @@ struct PkgConfig {
     }
     
     static func locatePCFile(name: String) throws -> String {
-        for path in (searchPaths + envSearchPaths) {
+        for path in (pkgConfigSearchPaths + searchPaths + envSearchPaths) {
```

Comment:
Isn't `pkgConfigSearchPaths` a superset of this, normally? Should we merge them?